build.gradle

```diff
@@ -113,9 +113,10 @@ subprojects { subproj ->
         maven { url = 'https://jitpack.io' }
         // Paper API
         maven { url = 'https://repo.papermc.io/repository/maven-public/' }
-        // Mojang Authlib
+        // Mojang, Authlib
```

Comment:
This repo is just for the Authlib dependency, I just typically call it "Mojang Authlib" to make it clear that Mojang is the author. Maybe there's a more clear way to say that but putting a comma between makes it seem like two separate things